in_toto/in_toto_mock.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+<Program Name>
+  in_toto_mock.py
+
+<Author>
+  Shikher Verma <root@shikherverma.com>
+
+<Started>
+  June 12, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Stripped down variant of in-toto-run command that can be used to mock metadata
+  generation of in-toto-run, without the need to specify a key and knowing all
+  the command line arguments.
```

Comment:
Maybe you want to mention how it differs from `in-toto-run` (what defaults it uses).